dataset/.gitignore

```diff
@@ -0,0 +1,6 @@
+/repos/
```

Comment:
what is the point of this ignore file?  any ignores necessary should be at the top level .gitignore, but also why are you ignoring files that are already in the repository?

Reply:
This is only for files ignored in this module of data generation.
Those ignored files are either temporary/generated input or output that should not be committed.
The number of those files are quite large, so ignoring them would help Git run much faster.